package.json

```
@@ -5,7 +5,7 @@
   "scripts": {
     "lint": "eslint .",
     "start": "./bin/cli.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "nyc mocha"
```

```suggestion
    "test:mocha": "mocha",
    "test": "nyc npm run test:mocha",
    "posttest": "npm run lint"
```

This makes it so that when you run `npm test` you get NYC coverage and it automatically runs the linter afterwards. It also provides a way to run unit tests without nyc for easier local development (`npm run test:mocha`, what you call this doesn't actually matter could be `test:unit` if you want).

---

Well that's a neat trick. Thanks for that one!